docs/gravitino-server-config.md

```diff
@@ -88,6 +88,50 @@ Gravitino server uses tree lock to ensure the consistency of the data. The tree
 
 Refer to [Iceberg REST catalog service](iceberg-rest-service.md) for configuration details.
 
+### Event listener configuration
+
+Gravitino provides event listener mechanism to allow users to capture the event provides by Gravitino server to integrate some custom operations.
```

Comment:
provides -> provided

Reply:
From your description, I still can't get what is `event`

Reply:
I add the description of the event after the configuration, is it enough?

Reply:
So, should you put it before the event listener?